src/components/common/ImportModal.tsx

```diff
@@ -156,11 +156,22 @@ export default function ImportSpeciesModal(props: ImportSpeciesModalProps): JSX.
 
   useEffect(() => {
     const getErrors = () => {
-      let errors = strings.DATA_IMPORT_FAILED;
-      if (fileStatus?.details.errors && fileStatus?.details.errors[0]) {
-        errors += ': ' + fileStatus?.details.errors[0].message;
-      }
-      return errors;
+      return (
+        <div className={classes.warningContent} key='import-error-1'>
```

Comment:
Not sure if I should have defined a new class for this. `warningContent` is just a left-align rule.

Reply:
This is fine